hovercraft/generate.py

```diff
@@ -107,13 +109,16 @@ def generate(args):
         template_info.add_resource(args.css, CSS_RESOURCE, target=target_path, extra_info='all')
         source_files.append(args.css)
 
-    # Make the resulting HTML
-    htmldata = rst2html(args.presentation, template_info, args.auto_console, args.skip_help,
-                        args.skip_notes)
+    # Make the resulting HTML (generation is relative to the basedir)
+    saved_cwd = os.getcwd()
+    os.chdir(args.basedir)
```

Comment:
Changing the current directory while regenerating the html is brittle, and means that if you reload the html while regenerating you get an error, I don't like that solution, we have to find something better.